demos/tests/args.py

```diff
@@ -50,6 +50,15 @@ def resolve(self, context):
         return str(context.dl_dir / context.model_info[self.name]["subdirectory"] / self.precision / (self.name + '.xml'))
 
 
+class ModelDirArg:
+    def __init__(self, model_name, file_name):
+        self.model_name = model_name
+        self.file_name = file_name
+
+    def resolve(self, context):
+        return str(context.dl_dir / context.model_info[self.model_name]["subdirectory"] / self.file_name)
```

Comment:
This wouldn't be a bad addition, but the model that required it no longer exists.